web/src/LogPane.test.tsx

```diff
@@ -404,9 +409,34 @@ it("renders highlighted lines", () => {
         handleClearHighlight={fakeHandleClearHighlight}
         highlight={highlight}
         modalIsOpen={false}
+        isSnapshot={false}
       />
     )
     .toJSON()
 
   expect(tree).toMatchSnapshot()
 })
+
+it("scrolls to highlighted lines in snapshot", () => {
```

Comment:
Relevant new test buried by large diff caused by my re-arranging the log fixture here.